tests/workspace_bridge/test_timeseries_plot.py

```
@@ -97,6 +97,22 @@ def test_downsample_aligned_plot():
 
 
+def test_evaluate_fault_plots_scopes_by_point_binding(plot_env: Path):
+    _reload_bridge()
+    from openfdd_bridge.feather_store import FeatherStore as FS  # noqa: E402
+    from openfdd_bridge.model_service import ModelService as MS  # noqa: E402
+    from openfdd_bridge.plot_readings import evaluate_fault_plots as eval_faults  # noqa: E402
+
+    model = MS().load()
+    df = FS().read_site("demo", source="bacnet")
+    assert df is not None
+    scoped_duct, panels_duct, _ = eval_faults(df, "demo", model, scope_columns=["duct-t"])
```

Ruff (0.15.15)

[warning] Unpacked variable `panels_duct` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)